app/javascript/tests/patient/Patient.test.js

```diff
@@ -74,6 +74,17 @@ describe('Patient', () => {
     });
   });
 
+  it('Properly renders identification section when patient is a minor', () => {
+    const wrapper = shallow(<Patient details={mockPatient5} hoh={mockPatient1} collapse={true} edit_mode={false} jurisdiction_paths={mockJurisdictionPaths} />);
+    const section = wrapper.find('#identification');
+    expect(section.find('h4').text()).toEqual('Identification');
+    expect(section.find('.edit-link').exists()).toBeTruthy();
+    identificationFields.forEach((field, index) => {
+      expect(section.find('b').at(index).text()).toEqual(field + ':');
+    });
+    expect(section.find('.text-danger').exists()).toBeTruthy();
```

Comment:
I would add the opposite of this check to the test before, as well as in this case check that the text is equal to `(Minor)`